config/manufacturer_specific.xml

```diff
@@ -939,7 +939,8 @@
 	</Manufacturer>
 	<Manufacturer id="0021" name="Zykronix">
 	</Manufacturer>
-	<Manufacturer id="0258" name="Neo Coolcam">
+	<Manufacturer id="0258" name="Neo CoolCam">
```

Comment:
Please always use latest version before do a PR. I move the Manufacturer in right order